### PR TITLE
Talent revision

### PR DESCRIPTION
**Pull request by milkavladislav**
- Button "Send invite to interview" should be blocked until all fields are filled (FE)
- If there's no job found, there should be a modal window. (FE)
- Remove "Budget" at Client's side (send invite) (FE + BE)
- Finish resently viewed freelancer logic (FE)

### CODE REVIEW

File: src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx [hunk 1]
```diff
@@ -79,44 +83,50 @@ function SubmitInviteModal({
 
   return (
     <Modal
-      title={`${firstName} ${lastName} ${t("Talent.inviteTitle")}`}
+      title={
+        jobPosts?.length === 0
+          ? "You don't have any posted jobs yet"
```
Comment (jphinning) [inline src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx]: Move the text to the localization file.

Also, please change the code to 
```typescript
jobPosts?.length ? "..." : "..."
```

File: src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx [hunk 2]
```diff
@@ -79,44 +83,50 @@ function SubmitInviteModal({
...
-            options={jobPosts}
-            label={t("Talent.jobs")}
-            control={control}
-            name="idJobPost"
-          />
-        )}
-        <SendButton>{t("Talent.sendButton")}</SendButton>
-      </Form>
+      {jobPostIsSuccess &&
+        (jobPosts.length === 0 ? (
```
Comment (jphinning) [inline src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx]: Here as well

File: src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx [hunk 3]
```diff
@@ -79,44 +83,50 @@ function SubmitInviteModal({
...
+          <Space size="large" direction="vertical">
+            <Text>Click below to post your first job</Text>
```
Comment (jphinning) [inline src/components/UI/ModalWindows/SubmitInviteModal/SubmitInviteModal.tsx]: Localization

File: src/pages/JobOwner/Talent/Index.tsx [hunk 4]
```diff
@@ -43,7 +43,7 @@ function Talent() {
     isLoading: isViewLoading,
   } = useGetRecentlyViewedFreelancersQuery();
 
-  const talentPart: ITalentPart[] = [
+  const talentParts: ITalentPart[] = [
```
Comment (lyudad) [inline src/pages/JobOwner/Talent/Index.tsx]: this constant have initialisation with each rerender, its very very bad practice

**Pull request merged**